eng/pipelines/extra-platforms/runtime-extra-platforms-ioslikesimulator.yml

```diff
@@ -58,10 +58,10 @@ jobs:
     isExtraPlatformsBuild: ${{ parameters.isExtraPlatformsBuild }}
     isiOSLikeOnlyBuild: ${{ parameters.isiOSLikeOnlyBuild }}
     platforms:
-      # - iossimulator_x64
-      # - tvossimulator_x64
-      # - ${{ if eq(variables['isRollingBuild'], true) }}:
-      - iossimulator_arm64
+      - iossimulator_x64
+      - tvossimulator_x64
+      - ${{ if eq(variables['isRollingBuild'], true) }}:
+        - iossimulator_arm64
```

Comment:
since this is not running on PR's by default, you can remove this condition.  Otherwise, it'll only ever run on the rolling build 2x per day and not on demand.

Reply:
Good catch. However, should it always test `iossimulator_arm64` as well?

Reply:
Yeah, I think so.